### ISSUE TITLE
How does this work?

### ISSUE THREAD

**Issue by shurik204** [Greazi-Times/Scoreboard-Number-Remover#1]
Can you explain how this works and how you came up with this idea?

**Comment by Greazi-Times** [comment]
De haven't made this for 100% my self.
I used some bits and part from a older version and made a pack out of it.

Ik hè idea came from the community of [TechsCode](https://techscode.com). I am a developer there where we create Minecraft plugins one of those is [Ultra Scoreboards](https://ultrascoreboards.com) an all in-game scoreboard editor.
A request of a pack that removes the scoreboard numbers has been made and there for I created a little pack that allows you to that just that for 1.17 and up.

**Comment by Greazi-Times** [comment]
De haven't made this for 100% my self.
I used some bits and part from a older version and made a pack out of it.

Ik hè idea came from the community of [TechsCode](https://techscode.com). I am a developer there where we create Minecraft plugins one of those is [Ultra Scoreboards](https://ultrascoreboards.com) an all in-game scoreboard editor.
A request of a pack that removes the scoreboard numbers has been made and there for I created a little pack that allows you to that just that for 1.17 and up.

**Comment by shurik204** [comment]
Interesting. You sure you made it? I've seen it elsewhere. 🤔
https://github.com/McTsts/mc-core-shaders/tree/main/hide%20sidebar%20numbers

**Comment by Greazi-Times** [comment]
Yes, I used that repo to get it to work it worked on 1.17 but not correctly on 1.18 I changed some things to get it to work.
I also have to say that I do not state anywhere that I made it 100% out on my own.

The only thing did I did is helping some people with a pre-made resource pack. 

But if it is that hard on your mind I can always remake the whole thing and on the end end up with the same thing.
